app/pathogen/arbovirus/layout.tsx

```diff
@@ -11,26 +16,14 @@ export default async function ArboLayout({
   const queryClient = getQueryClient();
 
   await queryClient.prefetchQuery({
-    queryKey: ["ArbovirusRecords"],
-    queryFn: () =>
-      fetch(`${process.env.NEXT_PUBLIC_API_BASE_URL}/arbo/records`, { cache: 'no-store' }).then(
-        (response) => response.json(),
-      ),
+    queryKey: ["arbovirusEstimatesQuery"],
+    queryFn: () => request(process.env.NEXT_PUBLIC_API_GRAPHQL_URL ?? '', arbovirusEstimatesQuery)
```

Comment:
@HimanshuRanka, if you're okay with the name could you please set `NEXT_PUBLIC_API_GRAPHQL_URL` to `https://iit-backend-v2.vercel.app/api/graphql` in all environments for me please? The vercel deployment won't work until you do hahahaha